### PR TITLE
Add map to Crash Details page

### LINKED ISSUE #85: VZE: Crash Details `/crash/:id`
- [x] details
- [x] drill down into related units, person, locations tables
- [x] map maybe?

### PR DESCRIPTION
**Pull request by mateoclarke**
Closes #85 

Screen Shot 2019-08-16 at 3 59 43 PM

### CODE REVIEW

File: atd-vze/package.json [hunk 1]
```diff
@@ -43,7 +44,7 @@
     "react-scripts": "^3.0.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "export HTTPS=true && react-scripts start",
```
Comment (mddilley) [inline atd-vze/package.json]: 🙌

**Comment by johnclary** [comment]
@mateoclarke can we get a screenshot????

**Comment by mateoclarke** [comment]
@johnclary added above...

**Comment by johnclary** [comment]
so awesome. love those flex bars underneath, too. y'all are making insane progress

**Comment by mateoclarke** [comment]
@mddilley I like `react-map-gl` ok. Its sweet that it has mapbox-gl with 3D tilt and panning but it might be overkill. Some decent examples out on the webs but so many more for Google Maps. I found myself popping back and forth between mapbox-gl docs and react-map-gl docs trying to figure out what things to change in the library vs the react wrapper. So a steeper learning curve but pretty powerful.

**Pull request merged**